components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -37,6 +37,7 @@ export default function OrganizationSelector() {
               title: currentOrg.data.name,
               customContent: (
                   <CurrentOrgEntry
```

Comment:
**thought**: We probably no longer need to separate the current entry as the styling is excaclty the same except from the checkmark icon used.